From 09b29e82c6641f76a695f4eac570d646e5eaf87b Mon Sep 17 00:00:00 2001
From: Dimitry Ishenko <dimitry.ishenko@gmail.com>
Date: Wed, 11 Sep 2024 00:05:24 -0400
Subject: [PATCH 1/4] Restore linking

Commit e0bbb969b171c85332e77ee8b50f29b4634b7ad7 changed how casparcg is
linked and launched on Linux, which caused a few problems.

It was partially reverted by ff1c0ba926c014f08e4c62af8e9c86013dd08a56
which restored how casparcg is launched, and this commit reverts the
linking part.
---
 src/shell/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/shell/CMakeLists.txt b/src/shell/CMakeLists.txt
index a7c485753..927ac8722 100644
--- a/src/shell/CMakeLists.txt
+++ b/src/shell/CMakeLists.txt
@@ -90,8 +90,6 @@ else ()
 		z
 		pthread
 	)
-
-    set_target_properties(casparcg PROPERTIES INSTALL_RPATH "$ORIGIN/../lib" BUILD_WITH_INSTALL_RPATH ON)
 endif ()
 
 add_custom_target(casparcg_copy_dependencies ALL)
-- 
2.43.0

